src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -59,15 +59,32 @@ protected EnergyNet(@Nonnull Location l) {
     public int getRange() {
         return RANGE;
     }
-
+    
+    /**
+     * @return An immutable {@link HashMap}
```

Comment:
Returns go after the description

Reply:
Oh I see, my bad uno momento

Reply:
Changed it! Also reworded a little bit to hopefully be easier to understand.